narwhals/expr.py

```diff
@@ -1130,7 +1130,12 @@ def arg_true(self) -> Self:
         """
         return self.__class__(lambda plx: self._call(plx).arg_true())
 
-    def fill_null(self, value: Any) -> Self:
+    def fill_null(
+        self,
+        value: Any | None = None,
+        strategy: Literal["forward", "backward"] | None = None,
+        limit: int | None = None,
+    ) -> Self:
```

Comment:
Just noticed, here and in `Series`, docstring should get updated as well to match the method arguments with their description.